lib/modules/manager/gleam/artifacts.ts

```diff
@@ -17,7 +17,7 @@ export async function updateArtifacts(
   const { packageFileName, updatedDeps, newPackageFileContent, config } =
     updateArtifact;
   logger.debug(`gleam.updateArtifacts(${packageFileName})`);
-  const isLockFileMaintenance = config.updateType === 'lockFileMaintenance';
+  const { isLockFileMaintenance } = config;
```

Comment:
When I see old code which references `config.updateType`, it implies to me that `config` just be an "update" config and not a "branch" config.

Reply:
no. it's the branch config, so it simply assumes the proper update type was copied to branch config